graphjet-core/src/main/java/com/twitter/graphjet/algorithms/counting/user/TopSecondDegreeByCountRequestForUser.java

```diff
@@ -33,6 +36,7 @@
   private final int maxNumSocialProofs;
   private final RecommendationType recommendationType = RecommendationType.USER;
   private final long maxEdgeEngagementAgeInMillis;
+  private final Set<Byte> socialProofTypeSet;
```

Comment:
prefer it.unimi.dsi.fastutil over java.util library.
interface: Set<Byte> => ByteSet
implementation: HashSet<Byte> => ByteOpenHashSet